nixos/modules/config/networking.nix

```diff
@@ -213,6 +213,10 @@ in
               otherHosts = allToString ( removeAttrs cfg.hosts [ "127.0.0.1" "::1" ]);
           in
           ''
+            127.0.0.1 ${cfg.hostName}
```

Comment:
It should be possible to disable this behavior.

Reply:
any ideas for the disable config option name?